contributor_docs/es/steward_guidelines.md

```diff
@@ -0,0 +1,211 @@
+
+
+##### Cerrando: No se puede Reproducir
+
+> No podemos reproducir esto, pero no dudes en reabrir si puedes proporcionar un ejemplo de código que demuestre el problema. ¡Gracias!
+
+
+##### Cerrando: Necesita Fragmento
+
+> Estoy cerrando esto por motivos organizativos. Por favor, reabre si puedes proporcionar un fragmento de código que ilustre el issue. ¡Gracias!
+
+
+##### Cerrando: Usa el Foro
+
+> Los issues de GitHub aquí son un buen lugar para los issues y problemas con la biblioteca p5.js en sí. Para preguntas sobre cómo escribir tu propio código, pruebas o seguir tutoriales, el [foro](https://discourse.processing.org/) es el mejor lugar para publicar. ¡Gracias!
+
+
+##### Cerrando: GSOC
+
+> ¡Gracias! El mejor lugar para discutir las propuestas de GSOC es en nuestro [foro](https://discourse.processing.org/c/summer-of-code).
+
+
+##### Cerrando: Acceso
+
+> No veo mucho interés en esta función, y no tenemos una explicación clara de cómo [amplía el acceso](access.md), así que cerraré esto por ahora. Si se puede agregar una declaración de acceso a la solicitud del issue, no dudes en volver a abrirlo.
+
+> No vemos una explicación más detallada de cómo esta cuestión [amplía el acceso](access.md), así que cerraré este issue por ahora. Si se puede agregar una declaración de acceso más detallada a la solicitud de función, no dudes en volver a abrirla. ¡Gracias!
+
+
+##### Cerrando: Complemento
+
+> Creo que esta función está fuera del alcance de la API de p5.js (intentamos mantenerla lo más minimalista posible), pero podría ser un gran punto de partida para una biblioteca complementaria. Consulta la documentación aquí sobre cómo crear un complemento: [https://github.com/processing/p5.js/blob/main/contributor\_docs/creating\_libraries.md](creating_libraries.md)
+
+
+##### Cerrando PR: Primero Necesita Issue
+
+> Gracias. Como recordatorio, primero deben abrirse issues antes de que se abran pull request y etiquetarse con el issue. Esto es necesario para realizar un seguimiento del desarrollo y mantener la discusión clara. ¡Gracias!
+
+
+##### Aprobar issue para corrección
+
+> Puedes seguir adelante con una solución. Gracias.
+
+
+##### PR Fusionado
+
+> Se ve bien. ¡Gracias!
+
+
+### GitHub CLI
+
+Revisar un PR complejo puede ser difícil con comandos de git complejos necesarios para obtener la versión del código del PR localmente para que puedas probarla. Afortunadamente,  [GitHub CLI](https://cli.github.com/) puede ayudar enormemente con este proceso y más.
+
+Después de instalar GitHub CLI e iniciar sesión, revisar una PR localmente se puede hacer ejecutando el comando `gh pr checkout [id_del_pull_request]`, y el proceso de obtener un <em>fork</em> remoto, crear una rama y cambiar a la rama se realizan automáticamente para ti. Volver a la rama principal será lo mismo que cambiar de rama ejecutando `git checkout main`. ¡Incluso puedes dejar un comentario en el PR desde la CLI sin necesidad de visitar la página web en absoluto!
```

Comment:
todas las palabras que se queden en ingles por favor ponerlas en itáliscas, esto aplica para todos los
- fork
- issues 
- pull requests